### ISSUE TITLE
rfc: test naming scheme

### ISSUE THREAD

**Issue by mischief** [coreos/mantle#154]
our test names are super disorganized, and have no consistency. having a proper naming scheme that is globbable by category would be nice.

something like:

```
base/adduser
fleet/submitunit
etcd/discovery
etcd/atomicswap
docker/push
docker/pull
systemd/journald/remote
systemd/nspawn
net/nfs/v3
net/nfs/v4
ext/deis
ext/kubernetes
```

then i can tell kola `run "docker/*"`, for example.

**Comment by mischief** [comment]
perhaps there ought to be another level, such as linux/base/adduser, linux/systemd/..., coreos/fleet/submitunit, coreos/etcd/atomicswap, coreos/rkt/install, ext/...etc.

**Comment by mischief** [comment]
fixed in #177/#178
